articles/confidential-computing/how-to-fortanix-confidential-computing-manager-node-agent.md

```diff
@@ -69,107 +90,126 @@ For Fortanix support, join the [Fortanix Slack community](https://fortanix.com/c
 
-You'll now create a token in Fortanix Confidential Computing Manager. This token allows a compute node in Azure to authenticate itself. Your Azure virtual machine will need this token.
+1. Select the **Infrastructure** → **Compute Nodes** menu item from the CCM left navigation bar and click the **+ ENROLL NODE** button.
+1. Click **COPY** to copy the Join Token. This Join Token is used by the compute node to authenticate itself.
 
-1. On the **Compute Nodes** tab, select **ENROLL NODE**.
-1. Select the **COPY** button to copy the join token. The compute node uses this join token to authenticate itself.
+   :::image type="content" source="media/how-to-fortanix-confidential-computing-manager-node-agent/nitro-join-token.png" alt-text="Screenshot that shows how to copy the join token.":::
 
-### Enroll nodes into Fortanix Node Agent
+### Enroll nodes into Fortanix Node Agent in Azure Marketplace
 
-Creating a Fortanix node agent will deploy a virtual machine, network interface, virtual network, network security group, and public IP address in your Azure resource group. Your Azure subscription will be billed hourly for the virtual machine. Before you create a Fortanix node agent, review the Azure [virtual machine pricing page](https://azure.microsoft.com/pricing/details/virtual-machines/linux/) for DCsv2-series. Delete any Azure resources that you're not using.
+Creating a Fortanix Node Agent will deploy a virtual machine, network interface, virtual network, network security group, and a public IP address into your Azure resource group. Your Azure subscription will be billed hourly for the virtual machine. Before you create a Fortanix Node Agent, review the Azure [virtual machine pricing page](https://azure.microsoft.com/pricing/details/virtual-machines/linux/) for DCsv2-Series. Delete Azure resources when not in use.
 
 1. Go to the [Azure Marketplace](https://azuremarketplace.microsoft.com/marketplace/) and sign in with your Azure credentials.
-1. In the search box, enter **Fortanix Confidential Computing Node Agent**. In the search results, select **Fortanix Confidential Computing Node Agent** to go to the [app's home page](https://azuremarketplace.microsoft.com/marketplace/apps/fortanix.rte_node_agent?tab=OverviewFortanix):
+1. In the search bar, type **Fortanix Confidential Computing Node Agent**. Select the App that shows up in the search box called **Fortanix Confidential Computing Node Agent** to go to the offering's home page.
 
-    :::image type="content" source="media/how-to-fortanix-confidential-computing-manager-node-agent/search-fortanix-marketplace.png" alt-text="Screenshot that shows how to get to the app's home page.":::
-1. Select **Get It Now**, provide your information if necessary, and then select **Continue**. You'll be redirected to the Azure portal.
+    ![search marketplace](media/how-to-fortanix-confidential-computing-manager-node-agent/search-fortanix-marketplace.png)
```

Comment:
```suggestion
    ![search marketplace](media/how-to-fortanix-confidential-computing-manager-node-agent/search-fortanix-marketplace.png)
```
can you update the alt text to follow the guidance? Something like "Screenshot that shows ... ." or "Diagram showing ... ."